File: qe_selenuim_python/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/login-form/")
print(driver.title)
driver.maximize_window()
time.sleep(1)

driver.find_element(By.CSS_SELECTOR, "#username").send_keys("admin")
driver.find_element(By.CSS_SELECTOR, "#password").send_keys("password")
driver.find_element(By.XPATH, "//button[text()='Submit']").click()

time.sleep(1)
print(driver.title)
driver.quit()

File: qe_selenium_java/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {

    public static void main(String[] args) throws InterruptedException {

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/login-form/");
        System.out.println(driver.getTitle());
        driver.manage().window().fullscreen();
        Thread.sleep(1000);

        driver.findElement(By.cssSelector("#username")).sendKeys("admin");
        driver.findElement(By.cssSelector("#password")).sendKeys("password");

        driver.findElement(By.xpath("//button[text()='Submit']")).click();

        Thread.sleep(1000);

        System.out.println(driver.getTitle());

        driver.quit();
    }
}